microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -153,56 +195,78 @@ The CIDR notation for grouping IP addresses was removed from the `clusterNetwork
+* Previously, the `microshift show-config` command did not show configuration changes. With this update, the behavior of the `microshift show-config` command shows the current configuration by default. (link:https://issues.redhat.com/browse/OCPBUGS-13221[*OCPBUGS-13221*])
 
-* Previously, the command `microshift show-config` was prevented from reporting the correct `memoryLimitMB` for etcd. With this update, `microshift show-config` reports the correct `memoryLimitMB` for etcd. (link:https://issues.redhat.com/browse/OCPBUGS-11734[*OCPBUGS-11734*])
+* Previously, KCM and KAS did not exit after getting a cancellation signal. With this update, {microshift-short} shuts down as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18773[*OCPBUGS-18773*])
 
-* Previously, OVN-Kubernetes failed to run on a disconnected {microshift-short} instance. With this update, the OVN-Kubernetes configuration supports running {microshift-short} on hosts with no default route. (link:https://issues.redhat.com/browse/OCPBUGS-11967[*OCPBUGS-11967*])
+* Previously, kubeconfigs used a certificate authority (CA) bundle that included 3 different signers: service network, localhost, and external. This made kubeconfigs interchangeable and capable of validating for other networks. With this update, the certificate authorities used to generate kubeconfig files for {microshift-short}'s embedded components were reconfigured to ensure the kubeconfigs are independent. (link:https://issues.redhat.com/browse/OCPBUGS-8301[*OCPBUGS-8301*])
+
+[discrete]
+[id="microshift-4-14-networking-bug-fixes"]
+==== Networking
+
+* Previously, mixing upper and lower case letters in the hostname makes {microshift-short} fail. This is because the node name was taken directly from the hostname, and Kubernetes does not allow upper case names. With this update, the hostname only uses lower case host names. (link:https://issues.redhat.com/browse/OCPBUGS-8411[*OCPBUGS-8411*])
 
 * Previously, the networking smoke test created a `hello-microshift` pod using `busybox` and an image pulled from docker. With this update, quay.io is used to host the image. (link:https://issues.redhat.com/browse/OCPBUGS-19939[*OCPBUGS-19939*])
 
+* Previously, when IP addresses were used instead of hostnames to connect to the apiserver, external connectivity failed. As a result, certificates did not regenerate when the IP address was changed. The kubeconfig used names instead of IP addresses which required the use of `subjectAltNames` if IP addresses were alternativly used. With this update, the previous kubeconfigs for absent IP addresses and names are removed. (link:https://issues.redhat.com/browse/OCPBUGS-11395[*OCPBUGS-11395*])
```

Comment:
```suggestion
* Previously, when IP addresses were used instead of hostnames to connect to the APIServer, external connectivity failed. The `kubeconfig` files used names instead of IP addresses which required the use of `subjectAltNames` if IP addresses were used instead of hostnames. As a result, CAs did not regenerate when an IP address was changed.  With this update, the previous `kubeconfigs` files for absent IP addresses and hostnames are removed. (link:https://issues.redhat.com/browse/OCPBUGS-11395[*OCPBUGS-11395*])
```